duckduckgo/Android
# ADS: Code and Attributes cleanup

Task/Issue URL: https://app.asana.com/0/1174433894299346/1203720277615468

### Description

### Steps to test this PR
Background
Task to track the work of removing unnecessary code in the app

Scope:
Attributes no longer in use
Remove old ButtonPrimaryRounded, ButtonSecondaryRounded, etc
Remove unused styles
